application/src/main/java/org/togetherjava/tjbot/commands/help/AskCommand.java

```diff
@@ -128,6 +129,7 @@ private RestAction<Message> handleEvent(InteractionHook eventHook, ThreadChannel
         helper.writeHelpThreadToDatabase(author, threadChannel);
         return sendInitialMessage(guild, threadChannel, author, title, category)
             .flatMap(any -> notifyUser(eventHook, threadChannel))
+            .flatMap(any -> pinFirstMessage(threadChannel))
```

Comment:
Undo all changes, and instead use this
```suggestion
        return sendInitialMessage(guild, threadChannel, author, title, category)
            .flatMap(Message::pin)
            .flatMap(any -> notifyUser(eventHook, threadChannel))
```

Reply:
lmao

Reply:
wow ok

Reply:
wait no this didn't worked

Reply:
![image](https://user-images.githubusercontent.com/73871477/194896828-748e2a8e-e6fc-49c5-ba42-8e61ca0a1c76.png)

Reply:
sounds like ur running it on the wrong message then. review the order of operations. maybe u have to move it one up or down or sth. i.e. right after the message from the thread is send.

Reply:
owh

Reply:
mb
```java
        return sendInitialMessage(guild, threadChannel, author, title, category)
            .flatMap(Message::pin)
            .flatMap(any -> notifyUser(eventHook, threadChannel))
            ```
            should work then

Reply:
![image](https://user-images.githubusercontent.com/73871477/194899670-9e0384fa-014c-4f77-8879-b35a8f6f77f3.png)

yes works again

Reply:
yes make a new one @Tais993